frontend/src/Connection.ts

```diff
@@ -97,10 +97,12 @@ export class Connection {
   private pingSent: Maybe<Types.Timestamp> = null;
   private resubscribeTo: Maybe<Types.ChainLabel> = getHashData().chain;
   private resubscribeSendFinality: boolean = getHashData().tab === 'consensus';
+  private updateThrottle = false;
```

Comment:
Just to clarify my understanding, is this used just to prevent calls to `window.requestAnimationFrame` while it is in the process of executing?

Reply:
Yup.